YAKL_parallel_for_fortran.h

```diff
@@ -581,37 +581,41 @@ namespace fortran {
     #endif
   }
 
-
-
-  template <class F> inline void parallel_for_cpu_serial( int &ubnd , F const &f ) {
-    for (int i0 = 1; i0 <= ubnd; i0++) {
+  template <class F> inline void parallel_for_cpu_serial( int ubnd , F const &f ) {
+    #pragma omp target teams distribute parallel for simd
```

Comment:
Wrap in `#ifdef __OPENMP45__` and all other instances.
Also, make sure I spelled that correctly here and in the other file.